tests/test_gaap.py

```diff
@@ -0,0 +1,183 @@
+#
+# LSST Data Management System
+# Copyright 2017 LSST/AURA.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+from __future__ import absolute_import, division, print_function
+
+import math
+import unittest
+import galsim
+import lsst.utils.tests
+import lsst.daf.base as dafBase
+import lsst.afw.detection as afwDetection
+import lsst.afw.geom as afwGeom
+import lsst.geom as geom
+import lsst.afw.table as afwTable
+import lsst.afw.image as afwImage
+import lsst.meas.base as measBase
+import lsst.afw.display as afwDisplay
+import lsst.meas.extensions.gaap
+
+
+try:
+    type(display)
+except NameError:
+    display = False
+    frame = 1
+
+
+def getGaapResultName(sF: float, sigma: float, name: str) -> str:
+    return "_".join((name, str(sF).replace(".", "_")+"x", str(sigma).replace(".", "_")))
```

Comment:
Please remove this duplication by making the version in the plugin more flexible (maybe make it a `classmethod`, and always have callers pass in the base name?)